provider/aws/lambda/autoscale/handler.go

```diff
@@ -354,5 +354,21 @@ func min(ii ...int64) int64 {
 }
 
 func main() {
+	sessionConfig := &aws.Config{Region: aws.String(os.Getenv("REGION"))}
+	session, err := session.NewSession(sessionConfig)
+	if err != nil {
+		fmt.Printf("Error creating new session: %+v", err)
+		return
+	}
```

Comment:
Change to use the `NewSession` because the `New` method is deprecated.

> session.New is deprecated: Use NewSession functions to create sessions instead. NewSession has the same functionality as New except an error can be returned when the func is called instead of waiting to receive an error until a request is made.  (SA1019)